### PR TITLE
Add WebGPU updates for Chrome 116

### PR DESCRIPTION
**Pull request by chrisdavidmills**
### Description

This PR adds some content for [Chromium 116 WebGPU updates](https://pr-6763-static-dot-dcc-staging.uc.r.appspot.com/blog/new-in-webgpu-116), namely:

- `GPUDevice.importExternalTexture()` and `GPUQueue.copyExternalImageToTexture()` can now use a `VideoFrame` object as a source. Details added to the intro and source description for each. Also added information on when each type of texture is destroyed (from `HTMLVideoElement`, and from `VideoFrame`) after being created with `GPUDevice.importExternalTexture()`. See https://gpuweb.github.io/gpuweb/#gpuexternaltexture

- `GPUAdapter.requestDevice()` now returns a device that is immediately lost, if it has already been used to create a device (i.e. `requestDevice()` was already called.) rather than the promise being rejected with `null`. You can then get lost info from it via `GPUDevice.lost`. This info has been added to the return value section.

- `GPUPipelineError.GPUPipelineError()` message argument is optional. Optional label added, as well as information on the default value.

### CODE REVIEW

File: files/en-us/web/api/gpuadapter/requestdevice/index.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ requestDevice(descriptor)
 
 A {{jsxref("Promise")}} that fulfills with a {{domxref("GPUDevice")}} object instance.
 
+If you make a duplicate call, i.e. call `requestDevice()` on a {{domxref("GPUAdapter")}} that `requestDevice()` was already called on, the promise fulfills with a device that is immediately lost. You can then get information on how the device was lost via {{domxref("GPUDevice.lost")}}.
+
+> **Note:** On older implementations, duplicate calls result in a promise that is rejected with `null`.
```
Comment (beaufortfrancois) [inline files/en-us/web/api/gpuadapter/requestdevice/index.md:44]: This is Chrome-specific.

**Review by beaufortfrancois — COMMENTED** [review]
LGTM with nit. Thank you!

File: files/en-us/web/api/gpuadapter/requestdevice/index.md (hunk 1 above)
Comment (chrisdavidmills) [reply files/en-us/web/api/gpuadapter/requestdevice/index.md]: Gotcha. I've removed it from the page. I don't think it is that vitally important to share, as it is early days and the implementations have converged now. But if anything, I might want to add it to the browser compat data. I'll see what the reviewers say when that is reviewed.

Comment (beaufortfrancois) [reply files/en-us/web/api/gpuadapter/requestdevice/index.md]: LGTM

**Review by Elchi3 — APPROVED** [review]
Thanks all! 👍

**Pull request merged**